Assets/Scripts/Scenes/InGame/Player/PlayerSpawn/PlayerSpawner.cs

```diff
@@ -13,11 +14,11 @@ public class PlayerSpawner : MonoBehaviour
         public static PlayerSpawner Instance;
         [Header("�v���C���[�̃X�|�[���֌W")]
         [SerializeField] private GameObject[] _myPlayerPrefab;//�v���C���[�v���n�u
-        [SerializeField] private GameObject[] _spawnPosition;//�v���C���[���X�|�[��������W
 
         [Header("UI")]
         [SerializeField] private GameObject _playerUI;
 
+        Vector3 SpawnPosition;
```

Comment:
```suggestion
private Vector3 _spawnPosition;
``` 
基本的にprivateを明記してください。また、命名規則に合わせてパスカルケースではなく_から始まるキャメルケースで書いてください。